index.html

```diff
@@ -21,15 +21,19 @@
       </ul>
     </div>
     <div id="content">
-      <h2 id="sdsTitle">Design a New Scratch Character</h2>
+      <h2 id="sdsTitle">10 Squares (The 100th SDS!)</h2>
       <p id="translatorCredit"></p>
       <p id="uniqueDescription">
-        Dream up a character for this SDS!<br/>
-        Create a new character that could be used as a sprite in someones project. Your character could be anything; an orange dragon, a fluffy cloud, or even a bag of popcorn with a face!<br/>
-        Once you're done, give your character a personality! Write a short description about it, it can be anything; what it likes, hobbies, favorite color, etc.<br/>
-        Be sure to animate your character in some way to give it its own twist!
-        <br/>
-        <br/>
+        Celebrate the 100th SDS with us by creating a project with 10 squares!
+        <br>
+        <br>
+Create a project that contains 10 squares on the screen at a time, and 10 squares only. It could be a train driving on a track, with the train made out of 7 squares, and the tracks made out of 3. It could be an Ice-cream simulator with the ice cream made out of 5 squares, and the ice cream machine made out of the other 5 squares. The possibilities are endless! 
```

Comment:
Please indent this line so that it is even with the other lines, just easier for editing in the future